inspectit-oce-agent/src/main/resources/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,2 @@
+Agent-Class: rocks.inspectit.oce.agent.main.Agent
```

Comment:
let's also right now add the:
```
Can-Redefine-Classes: true
Can-Retransform-Classes: true
```
just so that somebody does not break the head later on..